README.md

```diff
@@ -1,3 +1,5 @@
+[![Pylint](https://github.com/EECA-NZ/AZ-MDP-FUNC-EVROAM-LISTENER-REPO/actions/workflows/pylint.yml/badge.svg)](https://github.com/EECA-NZ/AZ-MDP-FUNC-EVROAM-LISTENER-REPO/actions/workflows/pylint.yml) [![Deploy](https://github.com/EECA-NZ/AZ-MDP-FUNC-EVROAM-LISTENER-REPO/actions/workflows/deploy-to-dev.yml/badge.svg)](https://github.com/EECA-NZ/AZ-MDP-FUNC-EVROAM-LISTENER-REPO/actions/workflows/deploy-to-dev.yml)
```

Comment:
Deploy badge URL doesn't exist. Maybe that's because it hasn't been created yet?

Also, do we want badges for prod deployment and unit tests? I don't badges add much unless it's an open source project. But could add them.